package.json

```diff
@@ -27,6 +27,18 @@
     "eslint-config-godaddy": "^4.0.1"
   },
   "bin": {
-    "orglinter": "./src/bin/cli.js"
-  }
+    "orglinter": "./bin/cli.js"
+  },
+  "keywords": [
+    "github",
+    "linting",
+    "organization-management"
```

Comment:
I'd do this as 2 separate keywords.

maybe also include things like "membership" "users" "synchronize" ?

Reply:
Awesome, thanks! I wasn't sure what the convention was in npm, so I just mirrored what's currently here on GitHub.

Reply:
this is basically your SEO keywords within NPM. typically you'd have it be the same as on github (because that's your SEO keywords for github as well), so maybe also update github with whatever you end up with.